app/src/org/commcare/tasks/FormLoaderTask.java

```diff
@@ -102,7 +102,7 @@ protected FECWrapper doTaskBackground(Uri... form) {
                     " from cached file: " + formBin.getAbsolutePath());
             fd = deserializeFormDef((Context)activity, formBin);
             if (fd == null) {
-                Logger.log(LogTypes.SOFT_ASSERT,
+                Logger.log(LogTypes.TYPE_RESOURCES,
```

Comment:
I would be in support of `LogTypes.SOFT_ASSERT` here since we are assuming that this line of code should never get called and `SOFT_ASSERT` is a way for us to validate that assumption.  I think with SUMO we should have some kind of alert setup  based on `SOFT_ASSERT` so that we get to know if a new `SOFT_ASSERT` is getting logged which most probably requires dev attention/action.

Reply:
mm that's a good point, I was still thinking along the lines of the `SOFT_ASSERT` tag not really being useful to us

Reply:
Sorry, I should amend that slightly-- that's a good point in general and I'm going to revert some of the other places in this PR where I removed a usage of `SOFT_ASSERT`. But this particular instance is one in which `SOFT_ASSERT` actually isn't right, because we do expect this to happen sometimes. (As the lines below indicate, we sometimes expect the form cache to be gone or invalid, and to have to re-load it from the file)